.github/CODEOWNERS

```
@@ -9,6 +9,7 @@
 # This is base configuration. These owners could review the
 # changes in all files in this repository.
 * @cognifloyd
+* @nzlosh
```

Based on https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file example the format is one-liner:
```suggestion
* @cognifloyd @nzlosh
```

Are you sure the multi-line format would work as expected (combined instead of overriding)?

---

Odd, my local git doesn't show this as a single line, but two, I'll try re-push it.

---

Looks to be fixed.

---

It's still the same in the last commit
https://github.com/StackStorm-Exchange/stackstorm-vault/blob/066eccbb47df89fcd8e6576d5fad47ccf8c6bed3/.github/CODEOWNERS#L11-L13

---

Just updated it in the https://github.com/StackStorm-Exchange/stackstorm-vault/pull/25/commits/aea76b211d0024752e123a7be1e92024a7e3fa3e

---

:man_facepalming: I don't know what happened to the CR.  Thanks for the fix.